package.json

```diff
@@ -41,6 +41,7 @@
     "@changesets/cli": "^2.11.0",
     "@spotify/eslint-config-oss": "^1.0.1",
     "@spotify/prettier-config": "^8.0.0",
+    "command-exists": "^1.2.9",
```

Comment:
Looks safe to add plus it is cross platform. [A good ~150 lines of helper code](https://github.com/mathisonian/command-exists/blob/master/lib/command-exists.js).

Feel free to suggest any alternative.